tox.ini

```diff
@@ -4,9 +4,25 @@ envlist = py27
 [testenv]
 deps = pytest
 install_command = pip install -e .[test] --find-links {toxinidir}/3rdparty {opts} {packages}
-# TODO(xujyan): Fix up 'test_scheduler_runs'.
 commands = py.test -v -k 'not test_scheduler_runs' tests {posargs:}
 
+[testenv:pex]
+deps =
+  pex
+  pytest
+  wheel
+whitelist_externals=mkdir
+commands =
+  mkdir -p {toxinidir}/dist/
+  pip install --find-links {toxinidir}/3rdparty -e .[driver]
```

Comment:
what are you trying to accomplish here?  shouldn't tox be doing the pip install?